libs/components/src/lib/tree-item/tree-item.scss

```
@@ -0,0 +1,104 @@
+@use "../focus/partials/variables" as focus-variables;
+
+@use "../../../../../dist/libs/tokens/scss/tokens.constants" as constants;
+
+@use "../../../../shared/src/lib/sass/mixins/connotation/config" with (
+	$connotations: accent,
+	$shades: primary primary-text primary-increment faint soft,
+	$default: accent
+);
+@use "../../../../shared/src/lib/sass/mixins/connotation" as connotation;
+
+@use "../../../../shared/src/lib/sass/mixins/appearance/config" as appearance-config with (
+	$appearances: ghost,
+	$states: idle disabled hover active selected selectedAndHover,
+	$default: ghost
+);
+@use "../../../../shared/src/lib/sass/mixins/appearance" as appearance;
+@use "../../../../../dist/libs/tokens/scss/size.variables" as size;
+
+@supports selector(:focus-visible) {
+	:host(:focus) {
+		outline: none;
+	}
+}
+
+.control {
+	@include appearance.appearance;
+	@include connotation.connotation;
+
+	$min-block-size: #{size.$vvd-size-normal};
+	$gap: 12px;
+	$gutter: 16px;
+
+	position: relative;
+	display: inline-flex;
+	box-sizing: border-box;
+	align-items: center;
+	background-color: var(#{appearance.get-appearance-token(fill)});
+	border-radius: 6px;
+	box-shadow: inset 0 0 0 1px var(#{appearance.get-appearance-token(outline)});
+	color: var(#{appearance.get-appearance-token(text)});
+	font: var(#{constants.$vvd-typography-base});
+	gap: $gap;
+	hyphens: auto;
+	inline-size: 100%;
+	min-block-size: $min-block-size;
+	padding-inline: $gutter;
+	text-decoration: none;
+	vertical-align: middle;
+	word-break: break-word;
+
+	@supports selector(:focus-visible) {
+		&:focus {
+			outline: none;
+		}
+	}
+
+	@supports (user-select: none) {
+		user-select: none;
+	}
+
+	&:not(.disabled) {
+		cursor: pointer;
+	}
+
+	&.disabled {
+		cursor: not-allowed;
+		pointer-events: none;
+	}
+
+	.text {
+		font: var(#{constants.$vvd-typography-base});
+	}
+}
+
+.expandCollapseButton {
+	position: relative;
+	display: flex;
+	align-items: center;
+	font-size: 20px;
+}
+
+.items {
+	display: flex;
+	flex-direction: column;
+	gap: 4px;
+	margin-block: 4px;
+	padding-inline: 60px;
```

```suggestion
	padding-inline-start: 48px;
```
(I also updated the value to align the start of the child at the end of the parent's expand/collapse indicator)